### ISSUE TITLE
Copy of Roll out npm audit checks to all repositories

### ISSUE THREAD

**Issue by nickaddy** [jac-uk/digital-platform#977]
Please see admin repo for reference.

We have a check for each PR and also a scheduled (daily) check to ensure there are no vulnerabilities in our code
`npm audit --omit=dev` I think is the command.

We need to roll out the same for all repos.

- [x] Admin
- [x] Apply
- [x] Assessments
- [x] QT
- [x] QT Admin

**Comment by nickaddy** [comment]
Can be rolled out to QT, but blocked on DP due to outstanding dependencies.

**Comment by KoWeiJAC** [comment]
The `npm audit checks` for `qt` and `qt-admin` have been set up.
<img width="1287" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeXJTQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1eb41f0df5cd28c8b4b03ab3335793b45c61d1f4/Screenshot%202024-01-17%20at%2009.12.06.png" alt="Screenshot 2024-01-17 at 09.12.06.png" />

**Comment by nickaddy** [comment]
@KoWeiJAC Pls close this ticket and create a new ticket for the remaining work which is currently blocked by the Firebase update. Move the new ticket to Blocked pipeline. Thank you.

**Comment by KoWeiJAC** [comment]
@nickaddy 
I've created 2 tickets for digital-platform and qt-platform:
- qt#56
- digital-platform#1009
